### PR TITLE
Fix ci script and add operator image environment created by openshift ci

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### What does this PR do?
Avoid to hardcoded in CI scripts the openshift CI image registry. I've added a new environment which get the operator image value from internal openshift registry and will use in tests

PR with openshift CI jobs tested: https://github.com/openshift/release/pull/12116

### CODE REVIEW

File: .cico/cico_openshift_e2e.sh [hunk 1]
```diff
@@ -43,9 +55,24 @@ if ! hash operator-sdk 2>/dev/null; then
         rm operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
 fi
 
+# Function to get all logs and events from devworkspace operator deployments
+function getCheClusterLogs() {
```
Comment (sleshchenko) [inline .cico/cico_openshift_e2e.sh:59]: CheCluster term usage here seems to be copy/paste mistake

Comment (flacatus) [reply .cico/cico_openshift_e2e.sh:59]: :O Right

**Review by sleshchenko — APPROVED** [review]
Thanks a lot for providing quick fixes for it!

File: .cico/cico_openshift_e2e.sh [hunk 2]
```diff
@@ -43,9 +55,24 @@ if ! hash operator-sdk 2>/dev/null; then
         rm operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
 fi
 
+# Function to get all logs and events from devworkspace operator deployments
+function getDevWorkspaceOperatorLogs() {
```
Comment (amisevsk) [inline .cico/cico_openshift_e2e.sh:59]: It's strange to have a function declaration this far down in the file (after some logic that is executed every time.

**Review by amisevsk — APPROVED** [review]
LGTM, minor comment. Thanks @flacatus

File: .cico/cico_openshift_e2e.sh [hunk 3]
```diff
@@ -7,24 +7,36 @@
...
+export NAMESPACE="devworkspace-controller"
 
 # Component is defined in Openshift CI job configuration. See: https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v4.yaml#L8
 export CI_COMPONENT="devworkspace-operator"
-# OPENSHIFT_BUILD_NAMESPACE env var exposed by Openshift CI. More info about how images are builded in Openshift CI: https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
-export IMG=default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/${OPENSHIFT_BUILD_NAMESPACE}/stable:${CI_COMPONENT}
+
+# DEVWORKSACE_OPERATOR env var exposed by Openshift CI in e2e test pod. More info about how images are builded in Openshift CI: https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
+# Dependencies environment are defined here: https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v5.yaml#L36-L38
+export IMG=${DEVWORKSACE_OPERATOR}
```
Comment (JPinkney) [inline .cico/cico_openshift_e2e.sh:39]: Should this be `DEVWORKSPACE_OPERATOR`? instead of `DEVWORKSACE_OPERATOR` (missing a p)

Comment (flacatus) [reply .cico/cico_openshift_e2e.sh:39]: FacePalm... My mistake :( :( https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v5.yaml#L36-L38
Need to change here

**Comment by flacatus** [comment]
@JPinkney @sleshchenko Provided fix here: https://github.com/openshift/release/pull/12128
Apologize for my mistakes :(

**Review by sleshchenko — APPROVED** [review]

**Pull request merged**